src/rsa/rsa_pkcs1_sign_tests.txt

```diff
@@ -460,3 +460,10 @@ Key = 308204a40201000282010100d3e94ceebdf732da0040d4ac1b3cc89b81718abfbb5661a516
 Msg = 0123
 Sig = b62bdd9c87f8f409178a5155768b536c1341f52a28c881494baa80b39f4a25a841f664eec49a464adf3cf5c920975cb83de4b06f0c7740685f0174627fcb4d0511bc6a25d473952c33fa5763a8e9bdd00bcdc1e4e5881d061193d77335d7e245987c9333455c9e1a2f0ff45242cff45cbd9ab8859d1c81dda62680ce388caba410be15bb40c0a66ea3a4ef07a3fb1f0f66c656c754dd3b15e694902ea79d1d0496a83408bdc2e94b7bdfef5883fe748ed50027ce0f35380dac8245e042ccdad69cd98440a2f77085a6d02d097f6ef650400b7eb469b9e6ee5284c7c7aee8e6365731114490dffadce428b2d16dd086ae0ef350f8e9dffe22d63203d891394322
 Result = Fail-Invalid-Key
+
+# Invalid key, p & q are too close
```

Comment:
If I calculated correctly, p - q == 348; i.e. they are extremely close together. Please include `p - q == 384` in the comment. Also, please add a comment that this is for a 4096-bit public modulus.

Please add a test vector for
https://gist.github.com/lgarron/44fb9af0e4ae922bce362b62de70e81b#file-2-nbits-div-2-100-apart. IIUC, this second test vector will test the case where they are almost far enough apart, but not quite.

Ideally, we'd also have a test vector for the case where they are just barely far enough apart.